tests/aws/services/sns/test_sns.py

```diff
@@ -3302,48 +3302,6 @@ def check_message():
 
         retry(check_message, retries=PUBLICATION_RETRIES, sleep=PUBLICATION_TIMEOUT)
 
-    @markers.aws.needs_fixing
-    # AWS validating this is hard because we need real credentials for a GCM/Apple mobile app
-    # Error responses are from reported https://github.com/spulec/moto/issues/2333
-    def test_create_platform_endpoint_check_idempotency(
```

Comment:
I wouldn't delete the test, even if validated in Moto. Does it still pass?

Reply:
Fair point about AWS validating the test. It asserts wrong behaviour so skipped for now. 

Added it back with some pointers :+1:

Reply:
Awesome, thanks! Yeah, Moto says the test has been AWS now, so I guess we can trust them, maybe he has some way of testing against AWS with an Apple developer account and fake app! We will need to get to it at some point. Thanks for putting it back!